lust/lib/models/capacity/percentPerHour.dart

```diff
@@ -0,0 +1,8 @@
+/// A class to describe the values in the capacity diagram.
+
+class PercentPerHour {
+  final String hour;
+  final int percent;
+
+  PercentPerHour(this.percent, this.hour);
```

Comment:
Maybe first hour and then percent. Since it is more reasonable to give the hour first and then the percent for that hour.

Reply:
Since it is percent per hour it is good in that order